resources/js/app.js

```diff
@@ -94,21 +94,213 @@
             controller: function ($scope, $location, Languages) {
                 self = this
 
+                $scope.message = {}
+                $scope.projectFilterOptions = {}
+                $scope.selectedStatusesList = []
+                $scope.selectedTagsList = []
+                $scope.selectedLevelsList = []
+                $scope.selectedInitiativesList = []
+                $scope.selectedCollabsList = []
+
                 var mapping = {
                     '': 0,
                     'crowded': 1,
                     'in_progress': 2,
```

Comment:
This is showing as `IN_PROGRESS` in the filter select list.

Reply:
Yes! I’m directly using these maps. Will change to remove that underscore in the filter option list

Reply:
I'm still seeing this - I tried a hard refresh in case it was cache related.

Reply:
Didn't solved this yet! Will solve this soon.